Src/PChecker/CheckerCore/SystematicTesting/ControlledRuntime.cs

```diff
@@ -460,6 +459,8 @@ private EnqueueStatus EnqueueEvent(ActorId targetId, Event e, Actor sender, Guid
                 "Cannot send event '{0}' to actor id '{1}' that is not bound to an actor instance.",
                 e.GetType().FullName, targetId.Value);
 
+            Scheduler.ScheduledOperation.LastSentReceiver = targetId.ToString();
```

Comment:
Can you please remind me why we need this?

Reply:
This information is used by POS algorithm to understand if two operations are conflicting each other.